lib/origin/selector.rb

```diff
@@ -168,7 +171,24 @@ def evolve_hash(serializer, value)
     #
     # @since 1.0.0
     def multi_selection?(key)
-      key =~ /\$and|\$or/
+      key =~ /\$and|\$or|\$nor/
+    end
+
+    # Determines if the selection operator is $in or $nin
+    # selection.
```

Comment:
You might want to describe this more generally, like 'Determines if the selection operator takes a list. Returns true for $in and $nin.'